src/ResourceGraph/ResourceGraph/ChangeLog.md

```diff
@@ -22,6 +22,7 @@
 
 ## Version 0.7.3
 * Improving logic of getting subscriptions for query to ARG
+* Updated package Microsoft.Azure.Management.ResourceGraph to version 2.0
```

Comment:
@nschonni it looks like you may have a couple of additional commits in this PR that can be resolved by pulling the latest from the `master` branch

Reply:
Yeah, I rebase with the latest master. Not sure what's happening with this one

Reply:
OK, fixed the rebase 😄